### PR TITLE
Replaced FileSizeFormat

### PR DESCRIPTION
**Pull request by scribe**

### CODE REVIEW

**Comment by scribe** [comment]
Don't merge yet, found a weird thing.

**Comment by shabtaisharon** [comment]
i don't know if anyone else wants to take a look at that, if you feel confident you can merge it :+1:

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java [hunk 1]
```diff
@@ -17,14 +17,18 @@
...
-public class ValueTreeTableCell<T> extends TreeTableCell<T,Number> {
+public class ValueTreeTableCell<T> extends TreeTableCell<T, Number> {
     @Override
     protected void updateItem(final Number item, final boolean empty) {
         super.updateItem(item, empty);
         if (empty || item == null) {
             setText(null);
         } else {
-            setText(FileSizeFormat.getFileSizeType(item.longValue()));
+            if (item.longValue() == 0L) {
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java]: I saw this same logic in another method.  Is this something that can be consolidated?

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/metadata/MetadataPresenter.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ private void initLabels() {
         metadataTableColName.setText(resourceBundle.getString("metadataTableColName"));
         metadataTableColValue.setText(resourceBundle.getString("metadataTableColValue"));
         objectName.setText(ds3Metadata.getName());
-        objectSize.setText(FileSizeFormat.getFileSizeType(ds3Metadata.getSize()));
+        objectSize.setText( FileSizeFormatKt.toByteRepresentation(ds3Metadata.getSize()));
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/metadata/MetadataPresenter.java]: There is an extra space before `FileSizeFormatKt`

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java (hunk 1 above)
Comment (scribe) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java]: Other method has been deleted.

**Pull request merged**